### ISSUE TITLE
hpcutil fails to connect on mentat005

### ISSUE THREAD

**Issue by robertoostenveld** [Donders-Institute/hpc-qaas#14]
```
roboos@mentat005> hpcutil webhook list
ERRO[0000] error retrieving webhook info from the QaaS server: 404 Not Found (HTTP CODE: 404) 
ERRO[0000] error retrieving webhook info from the QaaS server: 404 Not Found (HTTP CODE: 404)
```

Perhaps this is due to you not having completed the install yet. In that case, please do let me know when it is ready.

**Comment by aardkronkel** [comment]
Hong installed the latest version of the hpcutil on mentat999. Not yet on mentat005.

Can you access mentat999?

I will send a message when mentat005 is ready.

**Comment by hurngchunlee** [comment]
mentat005 is also installed.

**Comment by hurngchunlee** [comment]
But @rutgervandeelen  needs to fix an issue with the SSH command we just found out.  This issue can cause `qsub: command not found` error with some users (I am one of them).

**Comment by aardkronkel** [comment]
Just fixed it and made a new release 0.3.1. 

@robertoostenveld Does it work for you now?

**Comment by robertoostenveld** [comment]
yes, works now
